content/vi/docs/tutorials/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Tutorials
```

Comment:
Is this title right?

Reply:
Yes, I left it like that on purpose. When translating these words into Vietnamese, the meaning in the form of the title can be confusing. So I kept them as is, and explained what this document is in the overview of the index page.